### DISCUSSION TITLE
xSchedule API

### DISCUSSION THREAD

**Discussion by EDSTOBI** [xLightsSequencer/xLights#4888] [Q&A]
hello
Who can tell me the correct command to play my playlist 2024 with all the sequences it contains?
I would like to control the API via node red. When I test the commands directly in the browser, I always get the error {‘result’: ‘failed’, ‘command’: ‘Play selected playlist=2024’, ‘reference’:‘’, ‘message’: ‘Unknown command.’} with `http://localhost/xScheduleCommand?Command=Play%20selected%20playlist=2024`.

`http://localhost/xScheduleCommand?Command=Stop` works when I start it manually

Translated with DeepL.com (free version)

**Comment by keithsw1111** [comment]
http://localhost/xScheduleCommand?Command=Play%20specified%20playlist&Parameters=2024
